scripts/fetch-configlet

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+script_url="https://raw.githubusercontent.com/exercism/configlet/master/scripts/fetch-configlet-script"
```

Comment:
Should we rename the main branch from `master` to `main` in this repo and in `exercism/github-actions` in advance of doing it in all the other repos?

Reply:
We should definitely do it here. I'll update the PR. I don't know what happens if we rename the master branch on github-actions though. Does it still work when people use it with `@master`?

Reply:
> Does it still work when people use it with `@master`?

I think so. According to [this GitHub blog post](https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/):

> In addition, we have renamed the default branch for most of the GitHub-authored actions from `master` to `main`. All of your workflows that reference the old branch name will still work. However, you’ll see a prompt to change to the new name when you edit the workflow on the web.

But I think we can always try renaming from `master` to `main`, and if it stops working, temporarily create a `master` branch in `exercism/github-actions` that's even its `main`.

I notice that https://github.com/actions/checkout has both a `main` and `master` branch. But I don't know if this is why (it's also strange that `master` is 10 commits behind `main`...)

Reply:
See https://github.com/exercism/canonical-data-syncer/pull/140

Reply:
The rename of the github-actions is on our todo list BTW: https://github.com/exercism/v3/issues/3079#issue-792841783

Reply:
> But I don't know if this is why (it's also strange that master is 10 commits behind main...)

They switched the branch names before the renaming tool was available and probably kept master around to not break workflows that use `@master` for whatever reason.